setup_once.h

```diff
@@ -397,9 +397,9 @@ typedef int sig_atomic_t;
  */
 
 #ifdef DEBUGBUILD
-#define DEBUGF(x) x
+#define DEBUGF(channel, ...) ares_debug(channel, __VA_ARGS__)
```

Comment:
It certainly does! (I'm not sure how commonly used it is on such platforms of course)

If you build with `./configure --enable-debug` it may even warn if using features not C90 compatible.